### ISSUE TITLE
Keep file for reediting later

### ISSUE THREAD

**Issue by baby-gnu** [jlebon/textern#6]
Hello,

It's related to #5 to have stable filename and something I use with ITA, it permit to restore the content of the file when I make a mistake in the textarea.

Regards.

**Comment by jlebon** [comment]
Hmm, not sure. I really like the fact that all files are wiped as soon as possible. Note also that Textern itself exits and deletes the whole temporary directory itself when the last editor is closed.

The primary use case is to do all your editing in your main editor, where you (hopefully) have full undo history support. If you're doing final tweaks in the textarea directly after closing the editor, you can just Undo (Ctrl-Z) mistakes, right?

**Comment by baby-gnu** [comment]
Yes, but I had some uses, sometimes, of the way ITA works. You can close if you think you don't want it.

Regards.

**Comment by praiskup** [comment]
I also vote for this feature, it can be opt-in.  I lost my edits several times now (few minutes quite a large text).  The thing is that the browser isn't "locked", and when stupidly (a) click on some link and move to some other page and then (b) I save and close the editor, the last edit of the text is lost.
